zerver/lib/url_preview/preview.py

```diff
@@ -23,9 +23,15 @@
     r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'  # ...or ip
     r'(?::\d+)?'  # optional port
     r'(?:/?|[/?]\S+)$', re.IGNORECASE)
+
+# Use Chrome User-Agent, since some sites refuse to work on old browsers
+ZULIP_URL_PREVIEW_USER_AGENT = (
+    'Mozilla/5.0 AppleWebKit/537.36 (KHTML, like Gecko; compatible; ZulipURLPreview/{version}; '
+    '+{external_host}) Chrome/81.0.4044.113 Safari/537.36'
+).format(version=ZULIP_VERSION, external_host=settings.EXTERNAL_HOST)
+
 # FIXME: This header and timeout are not used by pyoembed, when trying to autodiscover!
```

Comment:
Merged.  But we may also want to look into this `FIXME`; we should either stop using `pyoembed` or figure out how to configure it to use this User-Agent as well.

Given how little code it is, I'd favor replacing it: https://github.com/mikeboers/PyOEmbed/blob/master/oembed.py

Reply:
I think this it the pyoembed that we use, which is also not super actively being maintained.
https://github.com/rafaelmartins/pyoembed/,